governance.md

```diff
@@ -0,0 +1,69 @@
+# Brigade Governance
+
+The following document describes how the Brigade project governance operates.
+
+## The Brigade Project
+
+The Brigade project is made up of several codebases with different release cycles. These codebases include:
+
+- Brigade
+- Kashti
+- gateways, libraries, and integrations
+
+There are two levels of maintainers for Brigade: org maintainers and project maintainers.
+
+## Brigade Org Maintainers
+
+The Brigade org maintainers are responsible for:
+
+- Maintaining the mission, vision, values, and scope of the project
+- Refining the governance and charter as needed
+- Making project level decisions
+- Resolving escalated project decisions when responsible project maintainers are blocked
+- Managing the Brigade brand
+- Managing access to Brigade assets such as source repositories, hosting, project calendars
+- Handling code of conduct violations
+- Deciding what sub-groups are part of the Brigade project
+- Overseeing the resolution and disclosure of security issues
+- Managing financial decisions related to the project
+
+The org maintainers have to match the following criteria:
+
+- There will be between 3 and 5 people with a diverse representation from multiple companies
+- Any project maintainer is eligible for a position as an org maintainer
+- No one company or organization can employ a simple majority of the org maintainers
+- Org maintainers MUST remain active on the project. If they are unresponsive for > 3 months, they will lose org maintainership, unless a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other org maintainers agrees to extend the period to be greater than 3 months
```

Comment:
"remain active on the project" has defining 'active' ever been an issue in Helm?

Reply:
Yeah. We have actually had to roll people off because they disappeared, there email started bouncing, etc. So we decided to add this because otherwise we had people with commit access, but with whom we quite literally could not communicate.

Reply:
Sorry, I should have clarified - has the definition of 'active' been an issue? E.g. is it code commits, Slack participation, Issue triage, community, etc..

Reply:
When "active" includes showing up at a meeting or being around on slack, it's only been an issue in one or two cases. And those were edge cases where people pretty much exited the entire Kubernetes community.